examples/helm-template/README.md

```diff
@@ -0,0 +1,36 @@
+# Helm Template
+
+This is an example of invoking the helm template function using declarative configuration.
```

Comment:
What does this do? What's the use case? Why do I care about this?

Reply:
Give higher level motivation now.